src/app/components/dashboard/dashboard.component.ts

```diff
@@ -10,117 +18,124 @@ import { ClientService } from '../../services/client.service';
 @Component({
   selector: 'app-dashboard',
   templateUrl: './dashboard.component.html',
-  styleUrls: ['./dashboard.component.scss']
+  styleUrls: ['./dashboard.component.scss'],
 })
-export class DashboardComponent implements AfterViewInit {
-
-  public address: string;
-
-  public clientInfo$: Observable<any>;
-  public chartData$: Observable<any>;
-
-  public chartOptions: any;
-
-  public networkInfo$: Observable<any>;
-
+export class DashboardComponent implements AfterViewInit, OnInit {
   @ViewChild('dataTable') dataTable!: Table;
 
-  public expandedRows$: Observable<any>;
+  private readonly autoRefresh$ = interval(600000).pipe(startWith(0));
```

Comment:
We shouldn't do auto refresh (at least default on) as it will hammer the api really hard as people leave it open on the browser. If you wanted to do this I would make a checkbox (default off) and increase the interval to 10min as it is the maximum resolution of the chart.